mat/triangular_test.go

```diff
@@ -513,7 +513,10 @@ func TestCopySymIntoTriangle(t *testing.T) {
 	}
 }
 
-func BenchmarkTriSum1000(b *testing.B) { triSumBench(b, 1000) }
+func BenchmarkTriSum1000(b *testing.B)      { triSumBench(b, 1000) }
```

Comment:
Since we're expanding the set of benchmarks, please use an iteration over some values and run sub-benchmarks.